vent/coordinator/process_manager.py

```diff
@@ -11,18 +11,16 @@ def __init__(self, sim_mode, startCommandLine=None, maxHeartbeatInterval=None):
         self.command_line = None  # TODO: what is this?
         self.max_heartbeat_interval = None
         self.previous_timestamp = None
+        self.child_process = None
         # TODO: if child process exists, need to reconnect it
-        self.child_process = multiprocessing.Process(target=rpc.rpc_server_main, args=(self.sim_mode,))
-        # TODO: when master process die, child process should survive
-        self.child_process.start()
-        self.child_pid = self.child_process.pid
+        self.start_process()
```

Comment:
shouldn't have multiple implementations of the same thing. for example, in this case the process was started with `daemon = False` in the `__init__` method, but started as a daemon in the `start_process` method.

Reply:
Agree with this. The daemon thing is something I didn't remove :D

Reply:
Forgot to remvoe*